src/composables/useAlerts.ts

```diff
@@ -0,0 +1,66 @@
+import { computed, ref } from 'vue';
+import { orderBy } from 'lodash';
+
+export enum AlertType {
+  ERROR = 'error',
+  INFO = 'info'
+}
+
+export enum AlertPriority {
+  LOW,
+  MEDIUM,
+  HIGH
```

Comment:
essentially these are just numbers (LOW = 0, MEDIUM = 1, HIGH = 2) - the range can be increased to accommodate for more priorities (and we can always just switching to using a number)